instagram-app/src/components/CommentSection/CommentAdd.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const CommentAdd = props =>{
+    return (
+        <div className="commentAdd">
+            <form onSubmit= {props.addComment}>
```

Comment:
don't think this is named the same when you are passing your props. Double check the naming!